src/lib.rs

```diff
@@ -195,6 +177,9 @@ pub trait PrimeField: Field + From<u64> {
     /// Returns the field characteristic; the modulus.
     fn char() -> Self::Repr;
```

Comment:
Let's remove this; the characteristic is not actually a valid field element, and we only ever want to operate over its bits.